src/processors/processVrmRequest.ts

```diff
@@ -0,0 +1,23 @@
+import { UserDetails } from '../services/user';
+import { TechrecordGet } from '../models/post';
+
+export const processPatchVrmRequest = (currentRecord: TechrecordGet, userDetails: UserDetails, newVrm: string): Array<TechrecordGet> => {
+  const recordToArchive: TechrecordGet = { ...currentRecord };
+  const newRecord: TechrecordGet = { ...currentRecord };
+  const date: string = new Date().toISOString();
+
+  if ('primaryVrm' in newRecord) {
+    newRecord.primaryVrm = newVrm.toUpperCase();
+  }
```

Comment:
```suggestion
    newRecord.primaryVrm = newVrm.toUpperCase();

```

Reply:
When I tried that I was getting a problem with the type:   Property 'primaryVrm' does not exist on type 'GETTRLTechnicalRecordV3Complete'.

Reply:
You can use `(newRecord as TechRecordHgv | TechRecordMotorcycle | TechRecordCar | TechRecordPsv).primaryVrm`
or the get schemas specifically